reference/sectionB_elements.inc

```diff
@@ -441,7 +441,7 @@ Table: Supported MathML Elements
 +----------------------------------+----------------------------------+
 | Calculus Elements                | <diff>                           |
 +----------------------------------+----------------------------------+
-| Qualifier Elements               | <bvar>, <logbase>, <degree> *    |
+| Qualifier Elements               | <bvar>, <logbase>, <degree> \*   |
```

Comment:
What's that star for?

Reply:
There's something under the table that was a note for this box?

Reply:
* (child of <root> or <diff>)